.zuul.d/jobs.yaml

```diff
@@ -193,3 +193,8 @@
 - job:
     name: ansible-runner-upload-container-image-stable-2.9
     parent: ansible-runner-container-image-base
+
+- job:
```

Comment:
https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/jobs.yaml#L441 is an example of how we do integration testing with ansible-runner.  You could do the same here.

Then adding a dependency on the ansible-runner-build-container-image like:

https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/network-jobs.yaml#L9

should get you the container on to the node to run tests against.

Reply:
@pabelanger First link seems out-of-date now. Any chance you could re-point me to the proper job?

Reply:
If you parent to ansible-buildset-registry-consumer (like we did for ansible-ee-tests-base) you'll get for free the setting up of a node to use containers. We'll also pull down the proper image from the container-build jobs too

https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/jobs.yaml#L351

https://zuul-ci.org/docs/zuul-jobs/docker-image.html gives some information on the flow

Reply:
I see what you're driving at, but the issue here is that we really need two parents: one for the ansible-tox-py38, and another for image re-use. Zuul doesn't allow for that.

Going to have to put this work on hold for a bit since I think these integration tests that use a container for testing is fundamentally flawed anyway. They are pulling the `ansible-runner:devel` image from quay, rather than building a new image based on the current code.

Reply:
actually, zuul does support 2 parents. You can do something like: https://github.com/ansible/network-ee/blob/main/.zuul.d/jobs.yaml#L45 and https://github.com/ansible/network-ee/blob/main/.zuul.d/jobs.yaml#L80.

We can do it because of how job variants work in zuul.